### ISSUE TITLE
audio lang defaults to ja-JP

### ISSUE THREAD

**Issue by BlastBolt5** [crunchy-labs/crunchy-cli#207]
**Describe the bug**
when -a is set to en-US if there is more episodes in ja-JP it will also download those even tho they aren't in english
**To Reproduce**
Steps / command to reproduce the behavior:
```
$ crunchy-cli archive --skip-existing -a en-US -s en-US -o 'Anime {audio}/{series_name}/Season {season_number}/{series_name} - S{season_number}E{episode_number} - {title}.mkv' $url

```

**Expected behavior**
Should only download the english episodes

**Client (please complete the following information):**
 - OS: Ubuntu Server 22.04 LTS
 - Version: crunchy-cli 3.0.0-dev.13 (dc431a9 2023-04-25

**Additional context**
Happening with Mobile Suit Gundam: The Witch from Mercury

**Comment by BlastBolt5** [comment]
I just tested the commit build, worked perfect for the issue I was having, thank you!

**Comment by ByteDream** [comment]
Fixed in [v3.0.0-dev.14](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.0.0-dev.14)
